lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
```

Comment:
(Note to self: I'll defer acting on this change until more input from RSpec team. I don't have an opinion either way.)